osf-cas/templates/configmap.yaml

```diff
@@ -665,15 +672,15 @@ config/log4j2.xml: |-
           </AsyncRoot>
       </Loggers>
   </Configuration>
-services/cas.json: |-
+services/cas-203948234207100.json: |-
```

Comment:
The reason for this naming change is because newCAS will throw warning messages for each one if they are not named by following this format `[a-zA-Z0-9]+\-[0-9]+`.